books/refgram/src/from_scratch/core_to_complete/simple.md

```diff
@@ -55,6 +55,22 @@ vare
 poia varu mai
 poio varu mai
 ```
+Alternatively, the GI variables can be replaced with KI. This replaces the `vi-`
+particles with regular `ve/fe...etc` particles. This is because the arguments of
+varu are already predicates.
+```
+po varu ke ka ko be
+vare
+  va vari
+    ve ke
+    fa ka
+    vei
+  fe vari
+    ve bi ke
+    fa ko
```

Comment:
```suggestion
  via vari
    ve ke
    fa ka
    vei
  fie vari
    vie bi zu ke
    fa ko
```
- You still need `via/fie` on `vare`.
- `ve bi ke` would mean it is a proposition that is not the one represented by `ke`, which is not the same as negating proposition `ke`. We thus have to keep `vie` and add `zu` to unbox the proposition from `ke`.

I think from this last point that it may not be very useful to provide this alternative after all.